_includes/config/setup-cron_2.2_how-to.md

```diff
@@ -29,17 +29,17 @@ crontab -l
 A sample follows:
 
 ```terminal
-#~ MAGENTO START
-* * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run | grep -v Ran jobs by schedule >> /var/www/html/magento2/var/log/magento.cron.log
-* * * * * /usr/bin/php /var/www/html/magento2/update/cron.php >> /var/www/html/magento2/var/log/update.cron.log
-* * * * * /usr/bin/php /var/www/html/magento2/bin/magento setup:cron:run >> /var/www/html/magento2/var/log/setup.cron.log
-#~ MAGENTO END
+#~ MAGENTO START c5f9e5ed71cceaabc4d4fd9b3e827a2b
+#* * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run 2>&1 | grep -v "Ran jobs by schedule" >> /var/www/html/magento2/var/log/magento.cron.log
```

Comment:
All crontab records are commented by `#` for some reason. I believe we should not do that if we are showing the working example.

Reply:
Good catch, @rogyar 

Thank you!